src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/components/step_time_field/components/time_field/time_field.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import {
+  EuiForm,
+  EuiFormRow,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonEmpty,
+  EuiSelect,
+  EuiText,
+  EuiLoadingSpinner,
+} from '@elastic/eui';
+
+export const TimeField = ({
+  isVisible,
+  fetchTimeFields,
+  timeFieldOptions,
+  isLoading,
+  selectedTimeField,
+  onTimeFieldChanged,
+}) => (
+  <EuiForm>
+    { isVisible ?
+      <EuiFormRow
+        label={
+          <EuiFlexGroup gutterSize="xs" justifyContent="spaceBetween" alignItems="center">
+            <EuiFlexItem grow={false}>
+              <span>Time Filter field name</span>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              { isLoading ? (
```

Comment:
Looks like there are a few spacing issues which cause jitter when switching between these two states:

![image](https://user-images.githubusercontent.com/1238659/35362228-60eb4cc2-0119-11e8-9fea-0d77fd115b7c.png)

We can fix this by changing the spinner to small (so it doesn't push the row taller), and then by using EuiLink and applying a class so its line-height doesn't do that either:

```jsx
              { isLoading ? (
                <EuiLoadingSpinner size="s"/>
              )
                : (
                  <EuiLink
                    className="timeFieldRefreshButton"
                    onClick={fetchTimeFields}
                  >
                    Refresh
                  </EuiLink>
                )
              }
```

```
/**
 * 1. Bring the line-height down or else this link expands its container when it becomes visible.
 */
 .timeFieldRefreshButton {
  line-height: 1 !important; /* 1 */
}
```